### ISSUE TITLE
facet browser: order properties in dropdown lists

### ISSUE THREAD

**Issue by mathib** [ali1k/ld-r#51]
Is it possible to order properties in the facet browser if you use the dropdown lists (categories)?

**Comment by ali1k** [comment]
in the fact configuration you can use the ldr:position property which would sort the selected properties. Have you tried that?

**Comment by mathib** [comment]
Yes, that works only if I don't use dropdown lists. I guess there should be something to order the dropdownlists and then per dropdownlist the properties of that dropdowlist.
